src/main/java/com/google/api/codegen/go/GoContextCommon.java

```diff
@@ -34,4 +35,34 @@
     }
     return result;
   }
+
+  public Iterable<String> getCommentLinesWrap(String text) {
+    List<String> result = new ArrayList<>();
+    for (String line : Splitter.on(String.format("%n")).split(text)) {
+      for (String wrapped : wrapLine(line, 70)) {
+        result.add(line.isEmpty() ? "//" : "// " + wrapped);
+      }
+    }
+    return result;
+  }
+
+  private List<String> wrapLine(String line, int length) {
+    if (line.length() <= length) {
+      return Collections.<String>singletonList(line);
+    }
+    List<String> result = new ArrayList<>(line.length() / length + 1);
+    StringBuilder current = new StringBuilder();
+    for (String word : Splitter.onPattern("\\s").omitEmptyStrings().split(line)) {
+      current.append(" ");
+      current.append(word);
+      if (current.length() >= length) {
```

Comment:
That would make things more intuitive, but it introduces the problem of having words that are by themselves longer than the limit. For this reason, I chose instead to go for a simpler algorithm that splits *after* the "limit" has been reached. We can solve both problems by using a more sophisticated algorithm, but I think this (at least for the time being) has more costs than benefits.

I have documented this in the Javadoc. Let me know if this works for you.